addons/web/static/src/js/views/form_relational_widgets.js

```diff
@@ -682,8 +671,17 @@ var AbstractManyField = common.AbstractField.extend({
         return command_list;
     },
 
+    _format_values : function (values) {
```

Comment:
keep previous name (`_format_many2one`) and clone() inside (or check if an util method exists to do the same)